### PR TITLE
stats: adjust CM sketch default size

### PR DESCRIPTION
**Pull request by lamxTyler**
If the values in cm sketch are too large, the encoded size may exceed max blob size 65535, thus result in write error.

### CODE REVIEW

File: statistics/cmsketch_test.go [hunk 1]
```diff
@@ -112,10 +113,16 @@ func (s *testStatisticsSuite) TestCMSketch(c *C) {
 }
 
 func (s *testStatisticsSuite) TestCMSketchCoding(c *C) {
-	lSketch, _, err := buildCMSketchAndMap(8, 2048, 0, 1000, 1000, 1.1)
```
Comment (hanfei1991) [inline statistics/cmsketch_test.go]: Why not use this function ?

Comment (lamxTyler) [reply statistics/cmsketch_test.go]: Because I need to construct the worst case, which all values are maximum.

**Comment by hanfei1991** [comment]
LGTM

**Pull request merged**